express/blocks/modal/modal.js

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable max-len */
```

Comment:
we don't need this. THis comes from milo now. And the migration of this is in a separate task cause we need to check that certain styles are overriden.

Reply:
I'd just remove for now and add a todo to revisit this in the gnav where we're importing.